services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_api.py

```diff
@@ -68,11 +70,11 @@ def dynamic_sidecar_settings(monkeypatch: MonkeyPatch) -> DynamicSidecarSettings
 
 
 @pytest.fixture
-def network_config(simcore_services_network_name: str) -> dict[str, Any]:
+def network_config(simcore_services_network_name: str, faker: Faker) -> dict[str, Any]:
     return {
         "Name": simcore_services_network_name,
         "Driver": "overlay",
-        "Labels": {"uuid": f"{uuid4()}"},
+        "Labels": {"uuid": f"{faker.uuid4()}"},
```

Comment:
minor and more of a question: not sure I get the point in this change. also does faker.uuid4() create a UUID or str?

Reply:
Yes, this change does not change the functionality but IMO sets the idea that you are faking some values.
IMO any value that is fake should go via ``faker`` fixture. Besides making it clear it allows us in the future to randomize those numbers (sometimes tests creates false negatives because of fixed unnoticed values)

Reply:
well I'm not sure I follow... uuid4 creates a UUID that is random.
faker.uuid4() creates a random string...
in this specific case I am not sure I see the point. I don't see the harm neither. But I think the intent was already clear and already random. (only this specific kind of cases).
Also, I find the cast part a bit cumbersome.

But hey that is just a thought... I'm ok with it ;)

Reply:
Yes, but my point is simply that it expresses better intent:

For instance, compare these two:
```python
def my_fixture_1() -> dict[str, Any]:
  return {  "uid": str(uuid4()), "number": 42 }
```
vs

```python
def my_fixture_2(faker: Faker) -> dict[str, Any]:
  return {  "uid": faker.uuid4(), "number": faker.pyint() }
```

I prefer the second

Reply:
yes I agree with 42... not the other. and still I don't like the implicit casting to str.